### DISCUSSION TITLE
Is there watermark support?

### DISCUSSION THREAD

**Discussion by tgermain-impirica** [diegomura/react-pdf#1990] [Q&A] [answered]
I would like to add a watermark across the pdf. Is this currently supported?

**Comment by CoskunCakir** [comment, marked as answer]
Hi @tgermain-impirica,

I believe you could do something like below to achieve this,

```jsx
<Text render={({ pageNumber, totalPages }) => `this is watermark`} fixed />
```

check the documentation's related section for this as well
https://react-pdf.org/advanced#dynamic-content

let me know how it goes,
cheers
